optax/_src/transform.py

```diff
@@ -594,6 +594,91 @@ def update_fn(updates, state, params=None):
   return base.GradientTransformation(init_fn, update_fn)
 
 
+class ScaleByAdanState(NamedTuple):
+  count: chex.Array
+  mu: base.Updates
+  nu: base.Updates
+  delta: base.Updates
+  grad_tm1: base.Updates
+
+
+def scale_by_adan(
+    b1: float = 0.98,
+    b2: float = 0.92,
+    b3: float = 0.99,
+    eps: float = 1e-8,
+    eps_root: float = 0.0,
+) -> base.GradientTransformation:
+  """Rescale updates according to the Adan algorithm.
+
+  References:
+    [Xie et al, 2022](https://arxiv.org/abs/2208.06677)
```

Comment:
same as before, wrong format for references.

please build the doc locally and make sure that all docstring display correctly